packages/fast-components/src/checkbox/checkbox.ts

```diff
@@ -38,6 +38,9 @@ export class Checkbox extends FormAssociated<HTMLInputElement> {
         this.defaultChecked = this.checkedAttribute;
     }
 
+    @attr
+    public disabled: boolean;
+

```

Comment:
@eljefe223 this seems unrelated; can we remove?

Reply:
Also, this is inheriting from form associated - if you were seeing an issue with disabled it was related to #2893 which has since been fixed.

Reply:
Ok, Didn't realize their was a bug tracking, adding this attribute fixed this but I can remove.